src/chrome/content/rules/Nintendolife.com.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Nonfunctional hosts in *.nintendolife.com:
+
+# All sites stay as http with this ruleset active.
+
+-->
+<ruleset name="Nintendolife.com">
+
+    <target host="nintendolife.com" />
+    <target host="www.nintendolife.com" />
+    <target host="3ds.nintendolife.com" />
+    <target host="3dsvc.nintendolife.com" />
+    <target host="3dsware.nintendolife.com" />
+    <target host="dsiware.nintendolife.com" />
+    <target host="ds.nintendolife.com" />
+    <target host="images.nintendolife.com" />
+    <target host="m.nintendolife.com" />
+    <target host="retro.nintendolife.com" />
+    <target host="static.nintendolife.com" />
+    <target host="vc.nintendolife.com" />
+    <target host="wii.nintendolife.com" />
+    <target host="wiiu.nintendolife.com" />
+    <target host="wiiware.nintendolife.com" />
+
```

Comment:
Add `<securecookie host=".+" name=".+" />` here.

Reply:
Can you explain what this does please?

Reply:
https://github.com/EFForg/https-everywhere/blob/master/docs/en_US/rulesets.md#secure-cookies